packages/surveys/src/utils/results-availability/migrate-form-properties-settings.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  getProp,
+  cloneObject,
+  IGetSurveyModelsResponse
+} from "@esri/hub-common";
+
+/**
+ * Populate form item properties is missing
+ * @param {IGetSurveyModelsResponse} _models
+ * @returns {IGetSurveyModelsResponse}
+ */
+export const migrateFormPropertiesSettings = (
+  _models: IGetSurveyModelsResponse
```

Comment:
I prefer to use `_` for the internal var, and no `_` for the external facing argument name.